src/embedded_flash/buffer_upgrade.rs

```diff
@@ -57,6 +57,10 @@ impl UpgradeStorage for BufferUpgradeStorage {
         }
     }
 
+    fn partition_address(&self) -> usize {
+        0x60000
```

Comment:
Is this address going to be generic enough for all boards?

Reply:
Generic in what sense? As in, if you have two physical partitions, the address is a tuple `(partition_name, address)`? If that's the case, we can always adjust to other boards later, if we need it?

Reply:
Generic in the way that the hardware abstraction is supposed to come from the kernel. An app should be able to work on any board provided all the syscalls it requires to work are implemented in the underlying kernel.
So I'm wondering about the relevance of a hard-coded address in an app.

Reply:
Oh, then we should be good. This is the test implementation, so `upgrade_locations` work in unit test in a `std` environment. The actual implementation is in the parallel file:
```
    fn partition_address(&self) -> usize {
        self.partition.start()
    }
```
And that values is read from syscalls.